docs/static/settings/monitoring-settings.asciidoc

```diff
@@ -25,8 +25,9 @@ the same {es} instance specified in the `outputs` section in your Logstash
 configuration, or a different one. This is *not* the URL of your dedicated
 monitoring cluster. Even if you are using a dedicated monitoring cluster, the
 Logstash metrics must be routed through your production cluster. You can specify
-a single host as a string, or specify multiple hosts as an array. Defaults to
-`http://localhost:9200`.
+a single host as a string, or specify multiple hosts as an array. Note that you
+should route Logstash metrics through coordinating nodes, and never through
```

Comment:
According to the Elasticsearch docs, even master-only nodes are "coordinating nodes".

> Every node is implicitly a coordinating node
> -- [Node docs: Coordinating Node](https://www.elastic.co/guide/en/elasticsearch/reference/current/modules-node.html#coordinating-node)

I _think_ I understand the intent here, which is to discourage those who have configured their clusters with dedicated master nodes from using resources on those dedicated masters, as doing so creates resource contention (the very thing they are trying to avoid when setting up dedicated master nodes in the first place).

Maybe something like a call-out box with:

~~~
NOTE: If your Elasticsearch cluster is configured with dedicated master-only nodes,
Logstash metrics should _not_ be routed to these nodes, as doing so can create
resource contention and affect the stability of the Elasticsearch cluster.
~~~

Reply:
Thanks for the feedback @yaauie. I have pushed an iteration along the lines of your suggestion.